protected/humhub/modules/friendship/Module.php

```diff
@@ -26,7 +26,7 @@ class Module extends \humhub\components\Module
      *
      * @return boolean is enabled
      */
-    public function getIsEnabled()
```

Comment:
@martin-rueegg It seems we should store this method as deprecated as you did for `isMarketplaceEnabled()`.

Reply:
Thank you @yurabakhtin for looking into this.

The issue is:
- `marketplace\Module` extends `humhub\components\Module`, whereas
- `friendship\Module` extends `\humhub\components\Module`

Now `\humhub\components\Module` newly defines `getIsEnabled()` to return the state if the _module_ is enabled. However, the old `friendship\Module::getIsEnabled()` returned whether the _functionality_ is enabled

So we have a real conflict here. Not an ideal situation. But as I've understood @luke- [here](https://github.com/humhub/humhub/pull/6550#issuecomment-1832191248), that was acceptable.

Reply:
@martin-rueegg Ok, got it, thank you for the explanation.

Reply:
You're welcome.

Thanks for the review!